hydra-access-controls/lib/hydra/config.rb

```diff
@@ -22,13 +24,19 @@ def [] key
           permissions
         when :user_model
           user_model
+        when :id_to_resource_uri
+          id_to_resource_uri
         else
           raise "Unknown key #{key}"
       end
     end
 
     attr_reader :permissions
-    attr_accessor :user_model
+    attr_accessor :user_model, :id_to_resource_uri
```

Comment:
change this to attr_reader (because you already have a setter). Then add some docs about why you'd want to set it and that it expects a lambda